mama/c_cpp/src/c/entitlement/noop/noop.c

```diff
@@ -47,7 +47,7 @@ noopEntitlementBridge_destroy(mamaEntitlementBridge bridge)
 mama_status
 noopEntitlementBridge_init(entitlementBridge* bridge)
 {
-    mama_log (MAMA_LOG_LEVEL_WARN,
```

Comment:
I would have thought that building with the noop bridge makes that clear, but then again we've never used entitlements, so I'll take your word for it.  

If we wanted to build a "null" entitlement bridge as you suggest, is there any doc on how to do that?

It's not *that* big a deal, but since I'm in "spring cleaning" mode I'm just trying to tidy up.

Reply:
Unfortunately no we don't have a standalone document for that but it should just be a case of copying and pasting this one and changing its function and library names. Non-entitled (noop) builds are something that a lot of people get nervy about on the market data side - some are resistant to a library like this existing at all since it could make it easy for them to consume data without being compliant with the market data vendor, so that's why this is a warning.